Tictactoe.cs

```diff
@@ -12,30 +12,52 @@ public class Tictactoe
         private MessageHandler MessageHandler { get; set; }
         public PlayerModel Player1 { get; set; }
         public PlayerModel Player2 { get; set; }
+        public UserInputValidator InputValidator { get; set; }
 
         public Tictactoe(IOutputWriter outputWriter)
         {
-            CurrentPlayer = Player.X;
             Player1 = new PlayerModel();
             Player2 = new PlayerModel();
             MessageHandler = new MessageHandler(outputWriter);
+            InputValidator = new UserInputValidator();
+            
+            CurrentPlayer = Player.X;
+
+            InitializeGame();
         }
 
         public void InitializeGame()
```

Comment:
This might no longer need to be public?
Follow up question: How does a consumer know that creating a new Tictactoe will display text asking for input.